readme.md

```diff
@@ -1,56 +1,46 @@
 ### fastapi for to serve geodata
 
 
-https://fastapi.tiangolo.com/advanced/templates/
-https://pybit.es/articles/how-to-handle-environment-variables-in-python/
+Access postges db with Python Fastapi backend and render map using Leaflet.js. 
 
+### Table of Contents
 
+    Installation
+    Usage
+    API Documentation
+    License
 
+### Installation
 
-Access postges db with Python Fastapi backend and Leaflet.js. Use HTMX to minimize javascript complexity.
+```bash
+git clone https://github.com/thisAKcode/fastapi_streetart_map.git
+pip install -r requirements.txt
+cd app
+python dbpg2.py
```

Comment:
Here I get: 

```
ModuleNotFoundError: No module named 'geoalchemy2'
```

Is the requirements.txt file up2date?

Reply:
I am also missing `shapely` and `geojson` after the pip install step.

Reply:
I will fix it.